src/app/home/page.tsx

```diff
@@ -18,13 +138,11 @@ export default function Home() {
 			}}
 		>
 			<Box sx={{ position: "absolute", top: "20px", left: "20px" }}>
-				{/* todo: Insert the logo here */}
```

Comment:
Why was this changed. I think it may be used for those who have the todo extension where they can easily see what tasks need to be done using their vscode TODO extension. Not sure if that was the intended use case but we can maintain it as it was.

Reply:
Great catch! I think I accidentally removed it